src/autopas/containers/verletClusterLists/traversals/VerletClustersTraversal.h

```diff
@@ -44,139 +47,31 @@ class VerletClustersTraversal : public TraversalInterface,
     if (dataLayout != DataLayoutOption::soa) return;
 
     auto &clusterList = *VerletClustersTraversalInterface<Particle>::_verletClusterLists;
-
-    auto numClusters = clusterList.getNumClusters();
-    const auto &aosToSoaMap = clusterList.getClusterIndexMap();
-
-    _clusterSoAs.resize(numClusters);
-
-    const auto _clusterTraverseFunctor = [this, &aosToSoaMap](Particle *clusterStart, int clusterSize,
-                                                              std::vector<Particle *> &clusterNeighborList) {
-      auto currentClusterIndex = aosToSoaMap.at(clusterStart);
-      FullParticleCell<Particle> cell{};
-      cell.reserve(clusterSize);
-      for (int i = 0; i < clusterSize; i++) {
-        cell.addParticle(*(clusterStart + i));
-      }
-      SoA<typename Particle::SoAArraysType> &soa = _clusterSoAs[currentClusterIndex];
-      soa.resizeArrays(clusterSize);
-      _functor->SoALoader(cell, soa);
-    };
-
-    clusterList.template traverseClusters<true>(_clusterTraverseFunctor);
+    clusterList.loadParticlesIntoSoAs(_functor);
```

Comment:
why not just:

```
if (dataLayout = DataLayoutOption::soa) {
   VerletClustersTraversalInterface<Particle>::_verletClusterLists->loadParticlesIntoSoAs(_functor);
}
```
same for `endTraversal()`

Reply:
Good point! I think I changed that part too often and this resulted in this code.